<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlashingOrderClass</class>
 <widget class="QWidget" name="FlashingOrderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>闪电交易助手</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>30</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>买入快捷键 : CTRL +  =</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>80</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>卖出快捷键:  CTRL + 一</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_buy_quantity">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_sell_quantity">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>每次买入数量:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>每次卖出数量:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_fresh">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_save">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combox_price_level">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>240</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_price_level_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>交易价格:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="flashingorder.qrc"/>
 </resources>
 <connections/>
</ui>
